ui/src/components/Header.js

```diff
@@ -114,7 +114,7 @@ export default function Headers() {
         <div className="usa-nav-container">
           <div className="usa-navbar">
             <div className="usa-logo">
-              <em className="usa-logo__text">DocExtractor.gov</em>
+           <a href='/'><em className="usa-logo__text">DocExtractor.gov</em></a> 
```

Comment:
Just for readability, shouldn't this be indented in a bit more?

```suggestion
              <a href='/'><em className="usa-logo__text">DocExtractor.gov</em></a> 
```